src/zimscraperlib/zim/creator.py

```diff
@@ -20,14 +20,17 @@
 from __future__ import annotations
 
 import datetime
+import io
 import pathlib
 import re
 import weakref
 from collections.abc import Callable, Iterable
+from logging import DEBUG
```

Comment:
I like to import just `logging` so that there is no `DEBUG` variable around that might look like what it's not. `logging.DEBUG` is very clear instead